api_volontaria/apps/position/models.py

```diff
@@ -0,0 +1,184 @@
+from datetime import datetime, timedelta
+import pytz
+from babel.dates import format_date
+from django.conf import settings
+from django.db import models
+from django.db.models.signals import post_save, pre_delete
+from django.dispatch import receiver
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+from django.contrib.auth import get_user_model
+from dry_rest_permissions.generics import authenticated_users
+# from api_volontaria.email import EmailAPI
```

Comment:
Remove this comment since it's no more needed

Reply:
done